### PR TITLE
[IA-4943] update bioconductor, gatk, and AOU images

### PR DESCRIPTION
**Pull request by LizBaldo**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-4943

### What

- Updates the Terra images containing R (Bioconductor and GATK) to use bioconductor 3.19 that contains the latest 4.4 R version

### Why

- Updating R to 4.4 is a security request, as well as releasing the latest bioconductor to Terra

### What to test

- 

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

File: jenkins/gce-custom-images/create_gce_image.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ DAISY_IMAGE_TAG="release"
 #    2. select the image of interest, say, `cos-89-16108-403-22`
 #    3. run `gcloud compute images describe cos-89-16108-403-22 --project cos-cloud | grep selfLink`
 #    4. extract the segments starting with 'projects'
-BASE_IMAGE="projects/cos-cloud/global/images/cos-109-17800-66-65"
+BASE_IMAGE="projects/cos-cloud/global/images/cos-101-17162-463-16"
```
Comment (LizBaldo) [inline jenkins/gce-custom-images/create_gce_image.sh:47]: This is what fixed resuming the runtimes. I am making a ticket on the IA board now to upgrade to the latest COS and making sure the startup script is compatible by doing a docker restart jupyter server that will properly mount the persistent disk

Comment (LizBaldo) [reply jenkins/gce-custom-images/create_gce_image.sh:47]: TODO ticket is here: https://broadworkbench.atlassian.net/browse/IA-4967

**Review by yonghaoy — APPROVED** [review]

**Comment by LizBaldo** [comment]
@yonghaoy I think you meant to close this PR instead? https://github.com/DataBiosphere/leonardo/pull/4490

**Review by yonghaoy — APPROVED** [review]

**Review by Qi77Qi — APPROVED** [review]

**Comment by LizBaldo** [comment]
Verified that GPU runtimes are still working 👍

**Pull request merged**